ftw/simplelayout/browser/configure.zcml

```diff
@@ -37,6 +37,13 @@
         />
 
     <browser:page
+        for="*"
+        name="bock_container_redirect_to_parent"
```

Comment:
Typo `bock`.

Wouldn't it be easier to also name the view `redirect_to_parent` but register it for `plone.folder.interfaces.IFolder`? This would automatically use this view for all folderish content, so there is only one right name to put in the FTI XML. But do we want that?
Just an idea...